contrib/kubespray/doc/step-by-step-openpai.md

```diff
@@ -0,0 +1,51 @@
+#### Configuration prepare
+
+- [layout.yaml](../../../examples/cluster-configuration/layout.yaml)
+- [service.yaml](../../../examples/cluster-configuration/services-configuration.yaml)
+
```

Comment:
in addition to the example, you might also want to add a reference link for how-to make these configuration files.

Reply:
Done